pkg/app/app.go

```diff
@@ -276,6 +276,38 @@ func (a *App) ConfigMapRefs() map[reftracker.RefKey]struct{} {
 	return configMaps
 }
 
+func (a *App) isSafeToFallbackToNoopDelete() bool {
```

Comment:
`isSafeToFallbackToNoopDelete` the name indicates that we are first trying to delete and then falling back to noopDelete in case of an error. Maybe we can have something like `noopDeleteDueToTerminatingNamespaces`?

Reply:
That makes sense to me.